scripts/release.ts

```diff
@@ -0,0 +1,229 @@
+import { exec as _exec, spawn } from 'child_process';
+import { promisify } from 'util';
+import yargs, { ArgumentsCamelCase, Argv } from 'yargs';
+import { logger, setLogLevel, LOG_LEVEL } from '@hubspot/local-dev-lib/logger';
+import semver from 'semver';
+
+import { name as packageName, version as localVersion } from '../package.json';
+import { EXIT_CODES } from '../lib/enums/exitCodes';
+import { confirmPrompt } from '../lib/prompts/promptUtils';
+import { build } from './build';
+
+// TODO: Change to import when this is converted to TS
+const { uiLink, uiLine } = require('../lib/ui');
```

Comment:
Can this be changed since the file is TS now?

Reply:
These unfortunately have to stay as `require` until these have been _actually_ converted to TS (and use `export` syntax)